src/styles.ts

```diff
@@ -14,8 +14,10 @@ export const Colors = {
     BROWNISH_GRAY: '#696969',
     LIGHT_GRAY: '#DDDDDD',
     LIGHTER_GRAY: '#E6E6E6',
+    LIGHTER_GRAY_2: '#EEEEEE',
```

Comment:
I also introduced this, but I just replaced `LIGHTER_GRAY` with this color.

Reply:
great! I was running out of names 😄